### ISSUE TITLE
Metadata container vs Source of Truth consistency

### ISSUE THREAD

**Issue by aizuchi0** [anansi-project/rfcs#24]
Hello! First off, thank you so much for starting this. Comic book metadata has needed some freshening up for a while.
So, my question. Do you envision some sort of embedded link in the metadata container to its original source of truth, which will reside alongside the other metadata?
I mention this because metadata container obsolescence will always be with us, I think, and some mechanism for linking sources of truth with one another might facilitate keeping the data alive and accurate. I dunno. It looks like a problem with no clear answers.

**Comment by gotson** [comment]
Yes, if you check the data model in Google spreadsheet there is a placeholder for external ids containing an url.
